docs/support.md

```diff
@@ -0,0 +1,20 @@
+# Report an issue or request a feature
+
+| Project | Issue tracker | 
+|---------|--------------------|
+| Cirq | https://github.com/quantumlib/Cirq/issues | 
+| OpenFermion | https://github.com/quantumlib/OpenFermion/issues | 
+| ReCirq | https://github.com/quantumlib/ReCirq/issues |
+| qsim | https://github.com/quantumlib/qsim/issues | 
+| Tensorflow Quantum |https://github.com/tensorflow/quantum/issues |
+
+To report an issue or request a feature in our open-source projects, 
+first check if there is already an open issue for your bug report / 
+feature request at the links above. If not, file an issue by following 
+the provided template and/or contributing guidelines. Thanks for helping
+improve open-source quantum software!
+
+# Questions
+
+If you're not sure whether a certain behavior is buggy, or if you have other questions,
+please ask us in the [Cirq Gitter channel](https://gitter.im/cirqdev).
```

Comment:
Should we link to any email lists, such as cirq-dev?  Eventually we should probably link to QCS support email lists, but maybe not yet.

Reply:
I like this idea. I'll add a link to cirq-dev.